payloads/v9/channel.ts

```diff
@@ -0,0 +1,1047 @@
+	 */
+	format_type: StickerFormatType;
+}
+
+/**
+ * https://discord.com/developers/docs/resources/channel#message-object-message-sticker-format-types
+ */
+export const enum StickerFormatType {
+	PNG = 1,
+	APNG,
+	LOTTIE,
+}
+
+/**
+ * https://discord.com/developers/docs/resources/channel#followed-channel-object
+ */
+export interface APIFollowedChannel {
+	/**
+	 * Source channel id
+	 */
+	channel_id: Snowflake;
+	/**
+	 * Created target webhook id
+	 */
+	webhook_id: Snowflake;
+}
+
+/**
+ * https://discord.com/developers/docs/resources/channel#reaction-object-reaction-structure
+ */
+export interface APIReaction {
+	/**
+	 * Times this emoji has been used to react
+	 */
+	count: number;
+	/**
+	 * Whether the current user reacted using this emoji
+	 */
+	me: boolean;
+	/**
+	 * Emoji information
+	 *
+	 * See https://discord.com/developers/docs/resources/emoji#emoji-object
+	 */
+	emoji: APIPartialEmoji;
+}
+
+/**
+ * https://discord.com/developers/docs/resources/channel#overwrite-object-overwrite-structure
+ */
+export interface APIOverwrite {
+	/**
+	 * Role or user id
+	 */
+	id: Snowflake;
+	/**
+	 * Either 0 (role) or 1 (member)
+	 *
+	 * {@link OverwriteType}
+	 */
+	type: OverwriteType;
+	/**
+	 * Permission bit set
+	 *
+	 * See https://discord.com/developers/docs/topics/permissions#permissions-bitwise-permission-flags
+	 *
+	 * See https://en.wikipedia.org/wiki/Bit_field
+	 */
+	allow: Permissions;
+	/**
+	 * Permission bit set
+	 *
+	 * See https://discord.com/developers/docs/topics/permissions#permissions-bitwise-permission-flags
+	 *
+	 * See https://en.wikipedia.org/wiki/Bit_field
+	 */
+	deny: Permissions;
+}
+
+export const enum OverwriteType {
+	Role,
+	Member,
+}
+
+/**
+ * https://discord.com/developers/docs/resources/channel#thread-metadata-object-thread-metadata-structure
+ */
+export interface APIThreadMetadata {
+	/**
+	 * The archived state of the thread
+	 */
+	archived: boolean;
+	/**
+	 * The id of the member that last changed `archived`
+	 */
+	archiver_id?: Snowflake;
+	/**
+	 * The duration to wait before automatically archive the thread
+	 */
+	auto_archive_duration: ThreadAutoArchiveDuration;
+	/**
+	 * An ISO8601 timestamp for when `archived` was last changed
+	 */
+	archive_timestamp: string;
+	/**
+	 * The locked state of the thread
+	 */
+	locked?: boolean;
+}
+
+export type ThreadAutoArchiveDuration = 60 | 1440 | 4320 | 10080;
+
+/**
+ * https://discord.com/developers/docs/resources/channel#thread-member-object-thread-member-structure
+ */
+export interface APIThreadMember {
+	/**
+	 * The id of the thread
+	 */
+	id: Snowflake;
+	/**
+	 * The id of the member
+	 */
+	user_id: Snowflake;
+	/**
+	 * An ISO8601 timestamp for when the member last joined
+	 */
+	join_timestamp: string;
+	/**
+	 * Member flags combined as a bitfield
+	 *
+	 * See https://en.wikipedia.org/wiki/Bit_field
+	 *
+	 */
+	flags: ThreadMemberFlags;
+}
+
+/**
+ * Currently undocumented
+ */
```

Comment:
We do not include this for undocumented things, we keep them without a docstring